modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -121,6 +131,54 @@ const moveGhost = (
 
   ghostElm.style.width = (width - overflowX) + 'px';
   ghostElm.style.height = (height - overflowY) + 'px';
+
+  // Code needed for dragging CEF elements (specifically fixing TINY-8874)
+  // The idea behind the algorithm is that the user will start dragging the
+  // CEF element to the edge of the editor and that would cause scrolling.
+  // The way that happens is that the user will trigger a mousedown event,
+  // then a mousemove event until they reach the edge of the editor. Then
+  // no event triggers. That's when I set an interval to keep scrolling the editor.
+  // Once a new event triggers I clear the existing interval and set it back to none.
```

Comment:
How well does this handle 'strange' behavior, like losing focus in the middle of things? It's not out of the realm for someone to suddenly tab away.

Reply:
If they suddenly tab away then a new 'mouseup' event would be submitted and that is handled elsewhere

Reply:
If that's fired, then that's good.